package.json

```diff
@@ -35,9 +35,9 @@
     "view": "yarn api-explorer",
     "wipe": "rm -rf api spec",
     "build": "rm -Rf packages/*/lib && run-p -c build:*",
-    "build:cjs": "lerna exec --stream 'BABEL_ENV=build_cjs babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments'",
-    "build:es": "lerna exec --stream 'BABEL_ENV=build babel src --root-mode upward --out-dir lib/esm --source-maps --extensions .ts,.tsx --no-comments'",
-    "build:ts": "lerna exec --stream --sort 'tsc -b tsconfig.build.json'",
+    "build:cjs": "lerna exec --ignore @internal/* --stream 'BABEL_ENV=build_cjs babel src --root-mode upward --out-dir lib --source-maps --extensions .ts,.tsx --no-comments'",
```

Comment:
We have to have some sort of marker to filter unwanted workspaces. We can't use `private` because we have private workspaces in packages/ that need to be built. And we can't target just the packages/ folder because Lerna.

Moving away from root build scripts, and defining them in pacakges and using `--if-present` when running them would be the correct way to do this, I think.